src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,835 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-eq-null */
+/* eslint-disable one-var */
+/* eslint-disable vars-on-top */
+/*global window, define, ns */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * @example
+ * <div class="ui-calendar" id="calendar"></div>
+ *
+ * id="calendar" should be fixed.
```

Comment:
id="calendar" should be fixed??
It means, I can't change the id the component??

Reply:
Yes. It cannot be changed.
CalendarObject creates a unique id value.
There may be other methods, but this is the current stable method.